public/js/annotator-madison.js

```diff
@@ -280,15 +279,10 @@ $.extend(Annotator.Plugin.Madison.prototype, new Annotator.Plugin(), {
   addNoteLink: function (field, annotation) {
 
     //Add link to annotation
-    var annotationLink = $('<a></a>').attr('href', this.options.path + '#' + annotation.link).text('Copy Annotation Link').addClass('annotation-permalink');
-    var noteLink = $('<div class="annotation-link"></div>');
     var linkPath = this.options.origin + this.options.path + '#' + annotation.link;
```

Comment:
Since this is using the `href` attribute now, dropping `this.options.origin` from this would make it a little more portable and still works. I ran into an issue with serving the application from `localhost:8000` with `php artisan serve` here, but dropping origin fixed it.